lambdas/packages/psycopg2Layer/python/psycopg2/__init__.py

```diff
@@ -0,0 +1,131 @@
+"""A Python driver for PostgreSQL
```

Comment:
why are they so nested btw? `lambdas/packages/psycopg2Layer/python/psycopg2/__init__.py`?

remove `/psycopg2Layer/python/`?

Reply:
A zip file is required to create a Lambda Layer (the packages are also uploaded as a Lambda Layer, each library is one Layer for reusing purposes), the zip file must be in this format: python/libraryHere.py. I tested uploading with a zip file containing something/libraryHere.py and it did not work, docs here: https://docs.aws.amazon.com/lambda/latest/dg/python-package.html#python-package-dependencies-layers.

Reply:
sounds good! does removing only `/psycopg2Layer` work?

Reply:
Yes it works, I removed `/psycopg2Layer ` cin the latest commit